Lib/go/go.swg

```diff
@@ -426,57 +426,41 @@
 /* Strings.  */
 
 %typemap(gotype)
-	char *, char *&, char[ANY], char[],
-	signed char *, signed char *&, signed char[ANY], signed char[],
-	unsigned char *, unsigned char *&, unsigned char[ANY], unsigned char[]
-"string"
+	char *, char *&, char[ANY], char[] "string"
 
 /* Needed to avoid confusion with the way the go module handles
    references.  */
-%typemap(gotype) char&, unsigned char& "*byte"
```

Comment:
This part of the change doesn't seem right.  I understand the point about signed char *, but this is signed char&.  It's not being treated as a string, it's being treated as a pointer.